cordova-plugin-outline/android/java/org/outline/vpn/VpnTunnelService.java

```diff
@@ -233,8 +257,7 @@ private synchronized OutlinePlugin.ErrorCode startTunnel(
     final boolean remoteUdpForwardingEnabled =
         isAutoStart ? tunnelStore.isUdpSupported() : errorCode == OutlinePlugin.ErrorCode.NO_ERROR;
     try {
-      vpnTunnel.connectTunnel(proxyConfig.host, proxyConfig.port, proxyConfig.password,
-          proxyConfig.method, remoteUdpForwardingEnabled);
+      vpnTunnel.connectTunnel(client, remoteUdpForwardingEnabled);
```

Comment:
Note for future: at this point perhaps the UDP handler could be wrapped to handle the UDP unavailability.

Reply:
Added a TODO on `connectTunnel`